src/demo-app/baseline/baseline-demo.html

```diff
@@ -14,9 +14,12 @@
       <input mdInput placeholder="Input" value="Text Input">
     </md-form-field>
     | Text 5 |
-    <md-select placeholder="Select">
-      <md-option value="option">Option</md-option>
-    </md-select>
+    <md-form-field>
```

Comment:
I do like the consistency of having both `select` and `input` inside the same `form-field`; it makes it clear that the prefix, suffix, hint, and error belong to the form field. Very explicit, no magic. It's also more maintainable for us, since we don't have to forward along every `form-field` API through select over time. 

I don't feel incredibly strongly about it, though. @mmalerba up to you if you think it would be the same amount of work to do it the other way around; it would certainly make the migration easier